lean/main/elaboration.lean

```diff
@@ -173,6 +173,9 @@ elab "#findCElab " c:command : command => do
 /-!
 TODO: Maybe we should also add a mini project that demonstrates a
 non # style command aka a declaration, although nothing comes to mind right now.
+--  How about a `conjecture` declaration, similar to `lemma/theorem`, except that 
```

Comment:
@adomani can you add this idea as a "TODO" item like above?

Reply:
Done!

Reply:
I do by the way ot think this should be done as an elaborator, you can just write a macro that does this.